gpkit/feasibility.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+from .variables import Variable, VectorVariable
+from .substitution import get_constants
+from .varkey import VarKey
+from .posyarray import PosyArray
+
+
+def feasibility_model(model, flavour="max", varname=None, constants=None):
+    """Returns a new GP for the closest feasible point of the current GP.
```

Comment:
Things that are unclear from this docstring:

 - does Model need to correspond to a GP, or is an SP ok too?
 - This doesn't actually return a GP; it returns a tuple containing a bunch of things.
 - There are no *args or **kwargs even though they are mentioned in the docstring